contents/blog/pricing-lessons.md

```diff
@@ -0,0 +1,96 @@
+---
+date: 2022-03-17
+title: The pricing guide I never had
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+Imagine being able to grow twice as fast with a few hours of work. Changing the way you price has a real chance to get you there. 
+
+Here is a breakdown of the counterintuitive lesson I've learned as a cofounder at PostHog - the more we give away for free, and the less we ask from users on day 1, the _more_ money we make.
+
+## Don't let pricing get in the way of adoption
+
+Most software companies agonize over how their product is adopted yet when it comes to pricing, they default to charging a certain recurring amount per month. The only thing under deep consideration is how _much_... not _how_.
+
+Our revelation was realizing that, for us, pricing shouldn't get in the way of adoption, because we don't sell top down into companies.. our most successful paying customers get there as follows:
+
+* an individual engineer finds and implements the free product
+* a very small team starts using us
+* someone on that team needs a paid feature
+
+So, how did we get pricing out of the way?
+
+**A free for life tier** means users don't _ever_ have to part with cash (or user data) to use our product. We do this in two ways - (i) we offer an open source product with a huge range of features; (ii) we offer a little free usage on cloud for free, forever. This means we can get into production and demonstrate value to users who need to build a business case and can't just pop in their card.
+
+**Encourage users to upgrade**, since you need people to do exactly that! We encourage people to put in their credit card - by offering our paid features starting at $0 (with a generous amount of free usage) if they do so. This lets users evaluate the paid product very easily, and it lets them upgrade with no friction if they're heavy users. This exchange feels fair to us - it helps us figure out who is serious versus playing around. Our numbers reflect that users feel the same way - over half our users enter their card details.
```

Comment:
```suggestion
**Encourage users to upgrade**, since you need people to do exactly that! We encourage people to put in their credit card by offering our paid features starting at $0 (with a generous amount of free usage) if they do so. This lets users evaluate the paid product very easily, and upgrade with no friction if they're heavy users. This exchange feels fair to us - it helps us figure out who is serious versus playing around. Our numbers reflect that users feel the same way - over half our users enter their card details.
```

This doesn't need separating, it can be one sentence.